book/src/configuration.md

```diff
@@ -347,3 +347,58 @@ max-wrap = 25 # increase value to reduce forced mid-word wrapping
 max-indent-retain = 0
 wrap-indicator = ""  # set wrap-indicator to "" to hide it
 ```
+
+## Keymap config
+
+This section contains configuration that have to do with the behavior of some
+key bindings, but not with key bindings themselves. For keybindings, see [Key
+remapping].
+
+[Key remapping]: ./keymap.md
+
+### `[keys.supertab]` Section
+
+
+| Key        | Description | Default |
+|------------|-------------|---------|
+| `command` | If set to a command name, then when the cursor is in a position with non-whitespace to its left, instead of inserting a tab, it will run the given command. If there is only whitespace to the left, then it inserts a tab as normal. One can emulate a tabout behavior with this with the `move_parent_node_end` command. See [Tabout Example] | `None` |
+| `supercede_menu` | Normally, when a menu is on screen, such as when auto complete is triggered, the tab key is bound to cycling through the items. This means when menus are on screen, one cannot use the tab key to trigger the supertab command. If this option is set to true, the supertab command always takes precedence, which means one cannot use the tab key to cycle through menu items. One of the other bindings must be used instead, such as arrow keys or `C-n`/`C-p`. | `None` |
```

Comment:
Ideally I think this could be covered by #5505. You could having a binding for the completion menu:

```toml
[keys.menu.completion]
tab = "supertab" # default would be "move_next_item" or whatever that menu command would be called
```

Reply:
I agree, the ideal solution is just being able to rebind the key in the config, but this currently isn't possible. And at least for me, being able to have supertab take precedence is necessary for it to be useful at all while auto complete is on at the same time.